app/celery/tasks.py

```diff
@@ -418,3 +419,9 @@ def send_inbound_sms_to_service(self, inbound_sms_id, service_id):
                                service_id, inbound_api.url, e))
             except self.MaxRetriesExceededError:
                 current_app.logger.exception('Retry: send_inbound_sms_to_service has retried the max number of times')
+
+
+@notify_celery.task(bind=True, name='process-ses-result')
+@statsd(namespace="tasks")
+def process_ses_results(self, response):
+    process_ses_response(response)
```

Comment:
this should probably have some kind of retry logic - what happens when things go bad? 

i suppose the nice thing about putting this on the queue instead of posting to the app, is that the app staying up isn't a thing we need to worry about if they go onto the queue (sns retried if we 500'd before).

But if we have a problem (realistically, that problem would be DB connectivity) then it'd be good if this retried a few times.

Reply:
If you use a try block you can retry after a time period.

try
....
except SNSExecption as exc:
...         # Retry in 1 minute.
...         raise self.retry(countdown=60, exc=exc)